stdlib/public/core/BridgeObjectiveC.swift

```diff
@@ -85,6 +85,18 @@ public protocol _ObjectiveCBridgeable {
 
 #if _runtime(_ObjC)
 
+@available(macOS 9999, iOS 9999, tvOS 9999, watchOS 9999, *, deprecated)
```

Comment:
That's not actually valid syntax, and it's an error that the compiler isn't shouting at you.

Reply:
oh it probably is shouting at me, I didn't have time to try compiling it before going to lunch